features/_ephemeral/file.include/etc/systemd/system/ephemeral-cryptsetup.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Requires=systemd-repart.service
+After=systemd-repart.service
+DefaultDependencies=no
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/sbin/cryptsetup open --type plain --cipher aes-xts-plain64 --key-size 512 --key-file /dev/urandom /dev/disk/by-partlabel/EPHEMERAL ephemeral
+ExecStart=/usr/bin/dd if=/dev/zero of=/dev/mapper/ephemeral bs=1M count=1
+ExecStart=/usr/sbin/mkfs.ext4 -O '^has_journal' /dev/mapper/ephemeral
```

Comment:
Why do we need this option?

Reply:
We don't strictly need it, but this is a performance tuning measure. the disk get's discarded on each boot anyways so keeping a journal only adds unnecessary overhead and decreases write performance